File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.java
class Solution {
    public int solution(String myString, String pat) {

        String myString1 = myString.toLowerCase();
        String pat1 = pat.toLowerCase();
        boolean contain = myString1.contains(pat1);
        if(contain){
            return 1;
        }
        else{
            return 0;
        }

    }
}

File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.cs
using System;

public class Solution {
    public int solution(string myString, string pat) {
        String myString1 = myString.ToLower();
        String pat1 = pat.ToLower();
        bool contain = myString1.Contains(pat1);
        return contain ? 1 : 0;
    }
}